gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=9afb3ca688fc12c761a0e9e4321e4d24e977a4a8916c8a768b1fe05ddb4d6b66
```

Comment:
Please add the 7.5.1 sha here, to make sure we use the valid Gradle wrapper.

Reply:
@TimvdLippe Thanks, I have added `distributionSha256Sum`. It's strange that `gradlew wraper` didn't add it automatically.

P.S. But are you it really works? I tried to put just random number to `distributionSha256Sum` value, and gradle still worked...

Reply:
Our GitHub action should verify that we are using the correct version

Reply:
But both builds in github actions were green...
Apparently, the checksum was not verified.

<img width="793" alt="image" src="https://user-images.githubusercontent.com/279773/198519201-baefb9ee-062c-4362-b8ef-c430549f230b.png">

Reply:
This is supposed to be verified by https://github.com/mockito/mockito/blob/eb85518cef362e39788a7e030bab8ac78eca76c9/.github/workflows/ci.yml#L52

Maybe this verification only runs if a hash exists? Or did we misconfigure the action?

Reply:
Oh yes, that's right: the verification runs only if checksum exists.
I tried to enter a wrong checksum, and the build failed:
> Expected checksum: 'NOPE596b10cce501591e92f229816aa4046424f3b24d771751b06779d58WRONG'
>  Actual checksum: 'f6b8596b10cce501591e92f229816aa4046424f3b24d771751b06779d58c8ec4'

see https://github.com/mockito/mockito/pull/2778